support-frontend/app/services/stepfunctions/Client.scala

```diff
@@ -34,11 +34,17 @@ class Client(client: AWSStepFunctionsAsync, arn: StateMachineArn) extends SafeLo
   private def startExecution(arn: String, input: String, name: String)(implicit
       ec: ExecutionContext,
   ): Response[StartExecutionResult] = convertErrors {
+    val proposedName = name + "-" + System.nanoTime().toString
+    val validName =
+      if (proposedName.length <= 80)
+        proposedName
+      else
+        (System.nanoTime().toString + "-" + name).take(78) + "--"
```

Comment:
the dashes are just an extra clue that it's been truncated.  We can only use dash and underscore in the string